apps/users/views.py

```diff
@@ -142,13 +144,49 @@ def create(self, request, *args, **kwargs):
     def update(self, request, *args, **kwargs):
         return update_instance(self, request, *args, **kwargs)
 #==================================================================================================
+# Creating tokens manually
+def get_tokens_for_user(user):
+    refresh = RefreshToken.for_user(user)
+
+    profile_picture_url = None
+    if user.profile_picture_url:
+        profile_picture_url = user.profile_picture_url.url  
+
+    return {
+        'username': user.username,
+        'first name' : user.first_name,
+        'last_name' : user.last_name,
+        'email' : user.email,
+        'mobile' : user.mobile,
+        'Profile picture URL': profile_picture_url,
```

Comment:
Why this is not changed? Profile picture URL also there should be underscores. No spaces allowed.

Reply:
done